Package.swift

```diff
@@ -16,6 +17,7 @@ let package = Package(
         .package(url: "https://source.skip.tools/skip-ui.git", from: "1.29.3")
     ],
     targets: [
+        .target(name: "SwiftUI", dependencies: ["SkipFuseUI"]),
```

Comment:
Similar to my other comment, could just depend on "SkipSwiftUI" right? Then SkipFuseUI would be the "import on any platform" and SwiftUI would be the "import on Android only" equivalents.